aspnetcore/mvc/views/tag-helpers/built-in/script-tag-helper.md

```diff
@@ -17,43 +17,24 @@ The [Script Tag Helper](xref:Microsoft.AspNetCore.Mvc.TagHelpers.ScriptTagHelper
 
 The Script Tag Helper allows you to specify a CDN for the script file and a fallback when the CDN is not available. The Script Tag Helper provides the performance advantage of a CDN with the robustness of local hosting.
 
-The following Razor markup shows the `script` element of a layout file created with the ASP.NET Core web app template:
+The following Razor markup shows the `script` element with fallback resources:
 
-[!code-html[](link-tag-helper/sample/_Layout.cshtml?name=snippet2)]
-
-The following is similar to the rendered HTML from the preceding code (in a non-Development environment):
-
-[!code-csharp[](link-tag-helper/sample/HtmlPage2.html)]
-
-In the preceding code, the Script Tag Helper generated the second script ( `<script>  (window.jQuery || document.write(`) element, which tests for `window.jQuery`. If `window.jQuery` is not found, `document.write(` runs and creates a script 
+```HTML
+<script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js"
+        asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
+        asp-fallback-test="window.jQuery"
+        crossorigin="anonymous"
+        integrity="sha384-tsQFqpEReu7ZLhBV2VZlAu7zcOV+rXbYlF2cqB8txI/8aZajjp4Bqd+V6D5IgvKT">
+</script>
+```
 
 ## Commonly used Script Tag Helper attributes
 
 See [Script Tag Helper](xref:Microsoft.AspNetCore.Mvc.TagHelpers.ScriptTagHelper) for all the Script Tag Helper attributes, properties, and methods.
 
-### href
-
-Preferred address of the linked resource. The address is passed thought to the generated HTML in all cases.
-
-### asp-fallback-href
-
-The URL of a CSS stylesheet to fallback to in the case the primary URL fails.
-
-### asp-fallback-test-class
-
-The class name defined in the stylesheet to use for the fallback test. For more information, see <xref:Microsoft.AspNetCore.Mvc.TagHelpers.LinkTagHelper.FallbackTestClass>.
-
-### asp-fallback-test-property
-
-The CSS property name to use for the fallback test. For more information, see <xref:Microsoft.AspNetCore.Mvc.TagHelpers.LinkTagHelper.FallbackTestProperty>.
-
-### asp-fallback-test-value
-
-The CSS property value to use for the fallback test. For more information, see <xref:Microsoft.AspNetCore.Mvc.TagHelpers.LinkTagHelper.FallbackTestValue>.
-
-### asp-fallback-test-value
+### asp-fallback-test
```

Comment:
I think I'd also include `asp-fallback-src` in this list. It's shown in the snippet above and I think it's good for context.

Here's the content to save you 2 seconds, if you decide to go with it:

```
### asp-fallback-src

The URL of a Script tag to fallback to in the case the primary one fails. For more information, see <xref:Microsoft.AspNetCore.Mvc.TagHelpers.ScriptTagHelper.FallbackSrc>.
```